api/src/main/java/io/strimzi/api/kafka/model/template/KafkaClusterTemplate.java

```diff
@@ -199,6 +200,16 @@ public void setInitContainer(ContainerTemplate initContainer) {
         this.initContainer = initContainer;
     }
 
+    @Description("Template for Kafka Cluster certificate")
```

Comment:
Maybe this way better?

```suggestion
    @Description("Template for Secret with Kafka Cluster certificate public key")
```